lang/python/python3-speedtest-cli/Makefile

```diff
@@ -0,0 +1,34 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-speedtest-cli
+PKG_VERSION:=2.1.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=speedtest-cli
+PKG_HASH:=cf1d386222f94c324e3125ba9a0d187e46d4a13dca08c023bdb9a23096be2e54
+
+PKG_MAINTAINER:=Jaymin Patel <jem.patel@gmail.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-speedtest-cli
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Command line interface for testing internet bandwidth using speedtest.net
+  URL:=https://github.com/sivel/speedtest-cli
+  DEPENDS:=+python3-pkg-resources
```

Comment:
Please also add a dependency on python3 or python3-light. (Yes, this means you will have to investigate the package a bit to see if python3-light is enough or if it really needs the full python3 package.)

Why should this package depend on python3 or python3-light? python3-pkg-resources currently [depends](https://github.com/openwrt/packages/blob/f0390a40b9552de1ec6ec0b88f065803ad7463e7/lang/python/python3/files/python3-package-pkg-resources.mk#L15) on python3. Suppose speedtest-cli also requires the full python3 package.

Then suppose python3-pkg-resources was updated to depend on python3-light tomorrow (because it has changed or we did a deeper investigation of the package, and now only python3-light is necessary). This indirect change will break speedtest-cli for all new installs and it will not be obvious why it is broken.

Reply:
Yes, Updated dependencies on python3 packages